governance/votes/Executive vote - April 18, 2024.md

```diff
@@ -0,0 +1,151 @@
+- Call `push(uint256 amt)` on the the PSM_PAX_A_INPUT_CONDUIT_JAR contract with an amount of **84,211.27 USDP**, depositing the contents of the contract to the PSM and receiving an identical amount of Dai into the PSM_PAX_A_JAR contract.
+- Call `void()` on the PSM_PAX_A_JAR contract to deposit the Dai to the Surplus Buffer.
+- Set PSM-PAX-A Debt Ceiling to **0 DAI**.
+- Decrease Debt Ceiling of PSM_PAX_A back to **0 DAI**.
+
+This will have the following net results:
+
+- The PSM_PAX_A Debt Debt Ceiling remains at 0 DAI after the transaction has been executed.
+- Dai generated from PSM_PAX_A increases by **84,211.27 DAI**.
+- USDP balance of MCD_JOIN_PSM_PAX_A increases by **84,211.27 USDP**.
+- USDP balance of PSM_PAX_A_INPUT_CONDUIT_JAR decreases to **0 USDP**.
+- Surplus Buffer increases by **84,211.27 DAI**.
+
+### Trigger Spark Proxy Spell
+
+- **Authorization:** [Ecosystem Approval](https://forum.makerdao.com/t/apr-4-2024-proposed-changes-to-sparklend-for-upcoming-spell/24033/7), Governance Polls [1095](https://vote.makerdao.com/polling/QmfGV2vt), [1096](https://vote.makerdao.com/polling/QmSYZSCQ), [1097](https://vote.makerdao.com/polling/QmcRdMyA), [1098](https://vote.makerdao.com/polling/QmSh8gyC), [1099](https://vote.makerdao.com/polling/QmUhT32b), [1100](https://vote.makerdao.com/polling/QmZND8WW), [1104](https://vote.makerdao.com/polling/QmVXriiT), [1106](https://vote.makerdao.com/polling/QmVsKsGa)
+- **Proposal:** [Forum post "[Apr 4, 2024] Proposed Changes to SparkLend for Upcoming Spell"](https://forum.makerdao.com/t/apr-4-2024-proposed-changes-to-sparklend-for-upcoming-spell/24033)
+
+If this executive proposal passes, the SparkLend Proxy Spell on Ethereum and Gnosis at [0x3d1DD14Fa08163E7f64b0abf0F514f6276f50882](https://etherscan.io/address/0x3d1DD14Fa08163E7f64b0abf0F514f6276f50882) will be executed, containing the following items:
+
+#### [Ethereum+Gnosis] Upgrade Pool Implementation to Disable Flashloan into Borrow Feature
+
+This item upgrades the SparkLend pool implementation to disable the flashloan-to-borrow feature, reducing unnecessary risks and aligning with the latest Aave V3 release standards for enhanced security.
+
+####  [Gnosis] Onboard sxDAI
+
+This item proposes to onboard sxDAI (Savings xDAI), a yield-enhanced version of sDAI on the Gnosis Chain, as collateral.
+
+#### [Gnosis] Onboard EURe
+
+This item proposes to onboard EURe (Monerium EUR emoney) as collateral on the Gnosis Chain.
+
+#### [Gnosis] Onboard USDC
+
+This item proposes to onboard USDC (Circle USDC) as collateral on the Gnosis Chain.
+
+#### [Gnosis] Onboard USDT
+
+This item proposes to onboard USDT (Tether USDT) as collateral on the Gnosis Chain.
+
+#### [Gnosis] Refresh Existing Market Parameter
+
+This item updates Gnosis market parameters, including a significant adjustment of the xDAI interest rate model to align with current conditions and the introduction of sxDAI as collateral.
+
+#### [Gnosis] Update xDAI Oracle Parameter
```

Comment:
We would include parameters. This is a simple copy-paste from the poll text.

Reply:
Considering this is the SubDao proxy spell, I'm not entirely sure if this will be beneficial for the voters or if it might make the Executive Summary too lengthy and difficult to read. The onboarding parameters have been approved and are available for review in the forum thread. I'll include them this time and see how they are displayed in the voting portal

Reply:
We actually feel very strongly about this. This cannot execute without MKR voters supporting it. They are taking responsibility and potentially liability when they vote to execute the proposal. The least we can do is make sure they have the parameters they are voting to enact readily available so that they can verify the underlying code if they so wish.